<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Amit Kapadia</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PinpointWCS by the Chandra X-ray Center</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="verticalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>false</bool>
      </property>
      <property name="handleWidth">
       <number>7</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QStackedWidget" name="stackedWidget_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_1">
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="DropArea" name="dropLabel_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>DROP FITS IMAGE HERE</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="margin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="GraphicsView" name="graphicsView_1"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_3">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="DropArea" name="dropLabel_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>DROP EPO IMAGE HERE</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="margin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="GraphicsView" name="graphicsView_2"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCoordinates"/>
    <addaction name="actionImageAdjustments"/>
    <addaction name="actionWCSInfo"/>
    <addaction name="separator"/>
    <addaction name="actionCoordinate_Table"/>
    <addaction name="separator"/>
    <addaction name="actionDegrees"/>
    <addaction name="actionSexagesimal"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionAVM"/>
    <addaction name="actionDetailed_AVM"/>
    <addaction name="separator"/>
    <addaction name="actionXMP_Packet"/>
    <addaction name="separator"/>
    <addaction name="actionFITS_Image"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLinear_Stretch"/>
    <addaction name="actionLogarithm_Stretch"/>
    <addaction name="actionSquare_Root_Stretch"/>
    <addaction name="actionInverseHyperbolic_Sine_Stretch"/>
    <addaction name="actionPower_Stretch"/>
    <addaction name="separator"/>
    <addaction name="actionInvert"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_Clockwise"/>
    <addaction name="actionRotate_Counterclockwise"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_PinpointWCS"/>
    <addaction name="actionPinpointWCSHelp"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <addaction name="actionFit_Point"/>
    <addaction name="actionCentroid"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_in_DS9"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Workspace"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuImage"/>
   <addaction name="menuAdvanced"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCoordinates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Coordinates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionWCSInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>WCS Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionImageAdjustments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image Adjustments</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionLinear">
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="actionQuadratic">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quadratic</string>
   </property>
  </action>
  <action name="actionCubic">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cubic</string>
   </property>
  </action>
  <action name="actionAVM">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>AVM</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionFITS_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>FITS Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionLinear_Stretch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Linear Stretch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionLogarithm_Stretch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Logarithm Stretch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionSquare_Root_Stretch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Square Root Stretch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionInverseHyperbolic_Sine_Stretch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Inverse Hyperbolic Sine Stretch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionPower_Stretch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Power Stretch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Invert ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAbout_PinpointWCS">
   <property name="text">
    <string>About PinpointWCS</string>
   </property>
  </action>
  <action name="actionCoordinate_Table">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Coordinate Table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRotate_Clockwise">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Clockwise ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRotate_Counterclockwise">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Counterclockwise ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionCentroid">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Centroid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionFit_Point">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fit Next Point</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionDegrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Degrees</string>
   </property>
  </action>
  <action name="actionSexagesimal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sexagesimal</string>
   </property>
  </action>
  <action name="actionNew_Workspace">
   <property name="text">
    <string>New Workspace</string>
   </property>
  </action>
  <action name="actionOpen_in_DS9">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open in DS9</string>
   </property>
  </action>
  <action name="actionPinpointWCSHelp">
   <property name="text">
    <string>PinpointWCS Help</string>
   </property>
  </action>
  <action name="actionDetailed_AVM">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Detailed AVM</string>
   </property>
  </action>
  <action name="actionXMP_Packet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>XMP Packet</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <customwidgets>
  <customwidget>
   <class>DropArea</class>
   <extends>QLabel</extends>
   <header>DropArea.h</header>
  </customwidget>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>GraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
